bdb/count.c

```diff
@@ -300,7 +301,12 @@ int bdb_count(bdb_state_type *bdb_state, int *bdberr)
     int ret;
     BDB_READLOCK("bdb_count");
```

Comment:
Still need readlocks here right? Also bdb_count should only be called for complete tables right? Not subsets, like `select count(*) from t where a < 10`